quickstarts/spring-with-s2i.yaml

```diff
@@ -0,0 +1,208 @@
+        To view the build status of the Spring application:
+        
+        1. Hover over the icon on the bottom left quadrant of the **rest-http-example** deployment to see the build status in a tooltip.
+        
+        1. Click on the icon for quick access to the build log.  
+        
+            - You should be able to see the log stream of the **rest-http-example-1** build on the **Build Details** page.
+            
+            - The application and its dependencies will be built into a container image and pushed to the OpenShift container registry.
+
+      review:
+        failedTaskHelp: >-
+          This task isn’t verified yet. Try the task again.
+        instructions: |-
+          To verify the build is complete:
+
+          - Wait for the build to complete.  It may take a few minutes.  Do you see a **Complete** badge on the page header?
+
+      summary:
+        failed: Try the steps again.
+        success: Your build is complete.
+      title: View the build status
+    - description: >-
+        To view the associated code:
+        
+        1. In the navigation, click [Topology]{{highlight qs-nav-topology}}.
+        
+        1. The icon on the bottom right quadrant of the **rest-http-example** deployment represents the Git repo of the associated code.  
+        
+            - The icon shown can be for Bitbucket, GitHub, GitLab or generic Git.  
+          
+        1. Click on the icon to navigate to the associated Git repository.
+
+      review:
+        failedTaskHelp: >-
+          This task isn’t verified yet. Try the task again.
+        instructions: |-
+          Verify you can see the code associated with the sample app:
+          
+          - Was a new browser tab opened with https://github.com/snowdrop/rest-http-example ?
+
+          - Return to the browser tab running OpenShift.  
+
+
+      summary:
+        failed: Try the steps again.
+        success: You accessed the git repo associated with your deployment.
+      title: View the associated Git repo
+    - description: >-
+        To view the pod status:
+        
+            - Notice that the
+        **rest-http-example** deployment has a pod donut imposed on the
+        circle, representing the pod status (i.e. blue = running).  
+            - The color of the donut indicates
+        the pod status.  
+
+        1. Hover over the pod donut to see the pod status in a tooltip.
+      review:
+        failedTaskHelp: >-
+          This task isn’t verified yet. Try the task again.
+        instructions: |-
+          To verify you see the pod status:
+          
+          - Do you see the number of associated pods and their statuses?
+
+      summary:
+        failed: Try the steps again.
+        success: Your deployment has one running pod.
+      title: View the pod status
+
+    - description: >-
+        Since a Java builder image was used during the Import from Git flow, a Java icon is used by default in topology.  
```

Comment:
```suggestion
        Since a Java builder image was used during the **Import from Git** flow, a Java icon is used by default in Topology view.  
```